progman.c

```diff
@@ -5,7 +5,7 @@
 #include "serial.h"
 #include "motion_control.h"
 
-#define PROGRAM_MAX_SIZE (1024 * 3)
+#define PROGRAM_MAX_SIZE (2560)
```

Comment:
Where does this magic number come from. Why was it 1024 * 3? Is the 3 important?